h2o-docs/src/product/data-science/eif.rst

```diff
@@ -0,0 +1,124 @@
+.. _isoforestextended:
+
+Extended Isolation Forest
+-------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+Extended Isolation Forest algorithm generalizes its predecessor algorithm,
+the `Isolation Forest <if.html>`__. The original `Isolation Forest <if.html>`__ algorithm brings a
+brand new form of detection, although the algorithm suffers
+from bias coming from tree branching. Extension of the algorithm
+mitigates the bias by adjusting the branching,
+and the original algorithm becomes just a special case.
+
+The cause of the bias is that branching is defined by the similarity
+to BST. At each branching point the
+feature and the value are chosen; this introduces the
+bias since the branching point is parallel to one of the axes.
+The general case needs to define a random slope for each branching point.
+Instead of selecting the feature and value, it selects a random slope :math:`n` for
+the branching cut and a random intercept :math:`p`. The slope can
+be generated from :math:`\mathcal{N(0,1)` Gaussian distribution, and the
+intercept is generated from the uniform distribution with bounds coming
+from the sub-sample of data to be split. The branching criteria for the data
+splitting for a given point :math:`x` is as follows:
+
+.. math::
+    (x - p) � n < 0
+
+Tutorials and Blogs
+~~~~~~~~~~~~~~~~~~~
+
+The following tutorials are available that describe how to use Extended Isolation Forest. 
+
+- `Master's thesis: Anomaly detection using Extended Isolation Forest <https://dspace.cvut.cz/bitstream/handle/10467/87988/F8-DP-2020-Valenta-Adam-thesis.pdf?sequence=-1&isAllowed=y>`__: The thesis deals with anomaly detection algorithms with a focus on the Extended Isolation Forest algorithm including the implementation to the H2O-3 open-source Machine Learning platform.
+- `Extended Isolation Forest jupyter notebook created by the authors of the algorithm <https://github.com/sahandha/eif/blob/master/Notebooks/EIF.ipynb>`__: Describes how Extended Isolation Forest behaves compare to Isolation Forest.
+
+
+Defining an Extended Isolation Forest Model
```

Comment:
I've noticed that there a several parameters that are listed in the H2OExtendedIsolationForestEstimator python file that aren't listed in the eif.rst file: `col_sample_rate_change_per_level`, `col_sample_rate_per_tree`, `export_checkpoints_dir`, `max_depth`, `min_rows`, `score_each_iteration`, `score_tree_interval`, `stopping_metric`, `stopping_rounds`, & `stopping_tolerance`. Is there a reason for that?

Reply:
Definitely a bug, could you @valenad1 take a look, please?

Reply:
Fixed in: ff3dcb86a04ea08f4b6373fc1dcf2abcd751f13e